src/components/IconSet.vue

```diff
@@ -142,7 +154,10 @@
           @click="selectingMode = false"
         >
           Selecting Mode
-          <Icon icon="carbon:close" class="inline-block text-xl align-text-bottom" />
+          <Icon
+            icon="carbon:close"
+            class="inline-block text-xl align-text-bottom"
+          />
```

Comment:
I mean, most of the changes in this file are formatting changes, can you revert them so the PR would be clear on fixing only the necessary part

Reply:
oh, got it ! Sorry for the confusion, working on it